Rakefile

```diff
@@ -0,0 +1,2 @@
+require_relative 'config/application'
```

Comment:
I would recommend running `rake -T` from the console. That should show you all the available rake tasks you have by default.